package.json

```diff
@@ -8,12 +8,40 @@
     "build:app": "vuepress build docs",
     "build:lambda": "netlify-lambda build lambda",
     "docs:build": "vuepress build docs",
-    "postinstall": "netlify-lambda install"
+    "postinstall": "netlify-lambda install",
+    "test": "./node_modules/.bin/jest --silent"
   },
   "author": "Alan Woo (Bin Studio) <alan@bin.am>",
   "license": "MIT",
+  "jest": {
+    "moduleFileExtensions": [
+      "js",
+      "json",
+      "vue"
+    ],
+    "transform": {
+      ".*\\.(vue)$": "vue-jest",
+      "^.+\\.js$": "<rootDir>/node_modules/babel-jest"
+    },
+    "moduleNameMapper": {
+      "^@/(.*)$": "<rootDir>/docs/.vuepress/$1",
+      "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/__test__/mocks/FileMock.js"
```

Comment:
`"<rootDir>/__test__/mocks/FileMock.js"` should be `"<rootDir>/__tests__/mocks/FileMock.js"` (missing the `s`)

Reply:
fixed, i renamed __test__ with __tests__ and forgot change that line, Thanks!